### ISSUE TITLE
differential velocity aberration data missing

### ISSUE THREAD

**Issue by keatonb** [spacetelescope/astrocut#58]
Looking at TESSCut products, the data array includes columns 'POS_CORR1' and 'POS_CORR2' but all the values are zero.  For a typical TPF, these columns give the per-image pointing offsets relative to the WCS information, and they are important for matching source locations between TESS and other data sets.  Is it possible to include the correct differential velocity aberration data in the TESSCut products?  I have code that uses this information to analyze TPFs that I hope to use on TESSCut image arrays as well.

**Comment by scfleming** [comment]
Do these change frequently from frame-to-frame?  For TESSCut we base things like the WCS solution on a single frame since TESSCut does not go through each frame in the cutout and register each image to a singular WCS solution.  Would grabbing the values from the frame TESSCut gets other info from be sufficient, or would it cause more problems than it's worth if the values change and the header value only applies to one of the frames in the cutout?

**Comment by keatonb** [comment]
Yes these change frame-to-frame to track the pointing variations.  The WCS info varies between FFI images.  The 'POS_CORR1' and 'POS_CORR2' columns give the pointing offset relative to some reference frame in pixel columns and rows (at least this is my understanding, I just recently became familiar with these data columns).  I think the best solution would be to go through the FFIs for each sector/cam/ccd and record the WCS info once to separate files.  Then have TESSCut reference these to compute the DVA info.
